### PR TITLE
chore(term): enable selecting a workload from config store

### PR DESCRIPTION
**Pull request by Lou1415926**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/term/selector/selector.go [hunk 1]
```diff
@@ -960,6 +954,31 @@ func (s *ConfigSelector) Job(msg, help, app string) (string, error) {
 	return selectedJobName, nil
 }
 
+func (s *ConfigSelector) Workload(msg, help, app string) (string, error) {
```
Comment (paragbhingre) [inline internal/pkg/term/selector/selector.go:958]: Can we put comment to this public function? 
And where are we calling this function from?

File: internal/pkg/term/selector/errors.go [hunk 2]
```diff
@@ -13,3 +18,51 @@ var (
 	// ErrSubnetsNotFound is returned when no existing subnets are found.
 	ErrSubnetsNotFound = errors.New("no existing subnets found")
 )
+
+// ErrNoWorkloadInApp occurs when there is no workload in an application.
+type ErrNoWorkloadInApp struct {
```
Comment (efekarakus) [inline internal/pkg/term/selector/errors.go]: can we add unit tests for these errors 🥺

Comment (Lou1415926) [reply internal/pkg/term/selector/errors.go]: done!

File: internal/pkg/term/selector/selector.go (hunk 1 above)
Comment (Lou1415926) [reply internal/pkg/term/selector/selector.go:958]: > Can we put comment to this public function?

done!

> And where are we calling this function from?

we will need it in `storage init`

File: internal/pkg/term/selector/errors_test.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+package selector
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestErrNoWorkloadInApp_RecommendActions(t *testing.T) {
```
Comment (efekarakus) [inline internal/pkg/term/selector/errors_test.go:19]: oh we can have testcases for `_Error` as well

**Review by efekarakus — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**